.github/workflows/tests.yaml

```diff
@@ -0,0 +1,69 @@
+name: tests
+on: ["push", "pull_request", "workflow_dispatch"]
+
+jobs:
+  build-test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.18', '1.17', '1.16', '1.15', '1.14', '1.13']
+
+    name: basic-with-go-v${{ matrix.go }}"
```

Comment:
Looks like this is missing a leading `"`.

Reply:
Removed the double quote. Thanks.